libs/Microsoft.MixedReality.WebRTC/Interop/PeerConnectionInterop.cs

```diff
@@ -433,8 +428,10 @@ internal struct VideoDeviceConfiguration
         public delegate void PeerConnectionAudioFrameCallback(IntPtr userData, AudioFrame frame);
 
         [UnmanagedFunctionPointer(CallingConvention.StdCall, CharSet = CharSet.Ansi)]
-        public delegate void PeerConnectionSimpleStatsCallback(IntPtr userData, string statsType, IntPtr statsObject);
+        public delegate void PeerConnectionSimpleStatsCallback(IntPtr userData, IntPtr statsReport);
```

Comment:
Use `StatsReport(Handle)` instead of `IntPtr`?

Reply:
As said IRL this is for reverse PInvoke so needs to be IntPtr